crypto/machine.go

```diff
@@ -36,9 +37,18 @@ type OlmMachine struct {
 
 	AllowUnverifiedDevices bool
 
+	// AcceptVerificationFrom determines whether the machine will accept verification requests from this device.
+	AcceptVerificationFrom func(*DeviceIdentity) bool
```

Comment:
If this returns false, the transaction will be cancelled. Do you mean it could ie. return a second value that says whether it should be cancelled or simply ignored? When would we want to just ignore it?

Reply:
At least if you're ignoring a user, you wouldn't want to respond to verification requests at all. Not sure if to-device events are filtered from /sync when you're ignoring someone server-side, but an option to ignore requests locally would probably be good anyway

Reply:
Resolving as I see you did this already.